deformable_ompl/ComputeTrialSetupQuery[] queries
---
deformable_ompl/ComputeTrialSetupResult[] results
